DefaultCode.cs

```diff
@@ -135,7 +135,7 @@ internal static class DefaultCode
             + "IntSliderControl Amount3 = 0; // [0,100] Slider 3 Description\r\n"
             + "#endregion\r\n"
             + "\r\n"
-            + "protected override IDeviceImage OnCreateOutput(PaintDotNet.Direct2D1.IDeviceContext deviceContext)\r\n"
```

Comment:
If you are going to update this line of code, you must also update the corresponding line of code here:

https://github.com/BoltBait/CodeLab/blob/aa43217e8962699f38275e534b0277a813ec4d18/CodeLabConfigDialog.cs#L598

Reply:
Done